python-packages/contract_wrappers/src/zero_ex/contract_wrappers/__init__.py

```diff
@@ -21,7 +21,7 @@
 To replicate these examples, one can use the `0xorg/ganache-cli`:code: docker
 image, which comes with the 0x contracts pre-deployed.  To start it::
 
-    docker run docker run -d -p 8545:8545 0xorg/ganache-cli
+    docker run -d -p 8545:8545 -e SNAPSHOT_NAME=0x_ganache_snapshot-v3-beta 0xorg/ganache-cli
```

Comment:
Ah, can we rename the snapshot without the v3-beta suffix?

Reply:
Maybe open a follow-up issue so we can roll this out?

Reply:
Done.

Reply:
I had wanted to do that initially, but the ganache image wasn't ready for that yet, but now it is so I changed it.